extensions/mathsys.py

```
@@ -20,18 +20,16 @@
+    async def init(self, socket: æ.WebSocket) -> namespace: return self
     # CLASS -> LOAD
     def process(self, code: str) -> bool:
         self.code = code
-        self.validate = mathsys.validate(self.code)
-        return self.validate
-    # CLASS -> VIEW
-    def view(self) -> str:
-        return mathsys.view(self.code)
+        try:
+            mathsys.compile(self.code)
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)